pkgs/development/php-packages/openswoole/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, buildPecl, php, valgrind, pcre2 }:
+
+buildPecl {
+  pname = "openswoole";
+
+  version = "4.11.1";
+   sha256 = "sha256-Rhoa4ny86dwB3e86/1W30AlDGRUDYjK8RusquKF5Izg=";
```

Comment:
Why is this indented with 3 spaces when other things in the same scope/block is indented with two spaces?

Reply:
@etu Not intended, must be a small typo that creeped in from the manual squashing of the commits. Is this something you like to have changed? If so what should the right indentation be?

Reply:
I guess I would first make it consistent with the rest of the file.

Look at other php-packages and get some inspirations maybe?

Reply:
All fixed now.